tests/phpunit/tests/test-wpml-gutenberg-string-registration.php

```diff
@@ -143,6 +143,64 @@ public function it_registers_strings_and_set_location() {
 		$subject->register_strings( $post, $package );
 	}
 
+	/**
+	 * @test
+	 * @group wpmltm-3081
+	 */
+	public function it_registers_strings_and_set_wrap_tag() {
+		$post               = \Mockery::mock( 'WP_Post' );
+		$post->post_content = '<!-- wp:something -->post content<!-- /wp:something -->';
```

Comment:
@kagg-design I think this line is a bit misleading.

In theory, you can put whatever you want here, right? Because `gutenberg_parse_blocks` will overrides whatever you put in the post content.

If I'm right, wouldn't be clearer to have something like that instead?

`$post->post_content = 'post content is not relevant in this test';`

Reply:
@andreasciamanna sure, done in all test functions.